crates/red_knot_python_semantic/resources/mdtest/call/getattr_static.md

```diff
@@ -59,7 +59,7 @@ import sys
 reveal_type(inspect.getattr_static(sys, "platform"))  # revealed: LiteralString
 reveal_type(inspect.getattr_static(inspect, "getattr_static"))  # revealed: Literal[getattr_static]
 
-reveal_type(inspect.getattr_static(1, "real"))  # revealed: Literal[1]
+reveal_type(inspect.getattr_static(1, "real"))  # revealed: Literal[real]
```

Comment:
Would be easy to fix this (special case `int.real` similar to how we do in `Type::member`), but since we don't really depend on `static_member` / `getattr_static` anymore, I didn't think it would be particularly important.

Reply:
This actually seems like it makes our inference more accurate, since `int.real` is a descriptor:

```pycon
Python 3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import inspect
>>> inspect.getattr_static(1, "real")
<attribute 'real' of 'int' objects>
>>>
```

Reply:
You're right!